emscripten.py

```diff
@@ -2310,7 +2353,7 @@ def create_sending_wasm(invoke_funcs, forwarded_json, metadata):
   if shared.Settings.SAFE_HEAP:
     basic_funcs += ['segfault', 'alignfault']
 
-  basic_vars = ['DYNAMICTOP_PTR']
+  basic_vars = ['DYNAMICTOP_PTR', 'tempDoublePtr']
```

Comment:
why is that needed? for wasm hopefully we don't need it at all, so I'm surprised to see it here.